packages/suite-data/src/translations/list-unused.ts

```diff
@@ -2,7 +2,8 @@
 /* eslint-disable no-restricted-syntax */
```

Comment:
It looks like `/* eslint-disable @typescript-eslint/no-var-requires */` is not required anymore, since there are not `var` statements, probably a leftover from previous implementation,

Reply:
lets resolve this here https://github.com/trezor/trezor-suite/pull/5972

Reply:
solved